### PR TITLE
Address PR review comments related to callChoreo.

### PR DESCRIPTION
**Pull request by nipunsampath**
## Purpose
Address PR review comments on the PR https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/112

### CODE REVIEW

File: components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java [hunk 1]
```diff
@@ -530,16 +530,23 @@ private void handleExpiredToken() throws IOException, FrameworkException {
             }
         }
 
-        public void resolveConsumerKeySecrete() throws SecretManagementException {
+        /**
+         * Resolves the consumer key and the secret from the connection metadata. Consumer key and secret should be
+         * present in the connection metadata as a variable or as an alias, otherwise a SecretManagementException will
+         * be thrown.
+         *
+         * @throws SecretManagementException {@link SecretManagementException}
```
Comment (ThaminduDilshan) [inline components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java:538]:
```suggestion
         * @throws SecretManagementException {@link SecretManagementException}.
```

File: pom.xml [hunk 2]
```diff
@@ -526,5 +526,6 @@
             **/*Component.java,
             **/*Holder.java
         </sonar.coverage.exclusions>
+        <jsr311.api.version>1.1.1</jsr311.api.version>
```
Comment (ThaminduDilshan) [inline pom.xml:529]: Shall we move this above the sonar coverage rules so that all the versions are listed in a single block?

**Review by ThaminduDilshan — DISMISSED** [review]

File: pom.xml (hunk 2 above)
Comment (nipunsampath) [reply pom.xml]: Resolved by 82dbf0c1e232702704d16820175afc16e45a4ec2

File: components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java (hunk 1 above)
Comment (nipunsampath) [reply components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java]: Resolved by 82dbf0c1e232702704d16820175afc16e45a4ec2

**Comment by nipunsampath** [comment]
Closing the PR since the PR builder for the master is failing [1]. Will re-open once the the master branch is stable.

[1] https://github.com/wso2-extensions/identity-conditional-auth-functions/actions/runs/4911801691

**Pull request closed without merging**